src/components/Calendar/Calendar.cy.tsx

```diff
@@ -153,4 +155,21 @@ describe("<Calendar />", () => {
             })
         );
     });
+    it("navigates to day when grid is clicked", () => {
+        const todayShortDMY = testDate
+            .toLocaleString("en-GB", { day: "numeric", month: "short", year: "numeric" })
```

Comment:
What do you think about extracting these format options? I feel like it'll make it more readable, even though it's only used twice.

Reply:
Minor comment but you can also use `.toLocaleDateString` instead since you don't want the time.

Reply:
It seems like it is only used here and also other pre-existing examples like `tomorrowDMY` is also created locally in the cypress function so maybe it's better to keep it like this